server/migrations/25_update_data_stores_id.up.sql

```diff
@@ -0,0 +1,9 @@
+BEGIN;
+
+UPDATE data_stores
+SET id = 'current'
+WHERE id = (select id from data_stores order by created_at limit 1);
```

Comment:
Shouldn't we get the one with `is_default = true` ?

Reply:
🤦 sure! changing it!

Reply:
query updated to consider `is_default`